server/src/schema.graphql

```diff
@@ -0,0 +1,54 @@
+scalar DateTime
+
+type Query {
+    info: String!
+    feed(orderBy: LinkOrderByInput): Feed!
+}
+
+type Mutation {
+    post(url: String!, description: String!): Link!
+    signup(email: String!, password: String!, name: String!): AuthPayload
+    login(email: String!, password: String!): AuthPayload
+    vote(linkId: ID!): Vote
+}
+
+type Link {
+    id: ID!
+    description: String!
+    url: String!
+    postedBy: User
+    votes: [Vote!]!
+    createdAt: DateTime!
+}
+
+type User {
+    id: ID!
+    name: String!
+    email: String!
+    links: [Link!]!
+}
+
+type Vote {
+    id: ID!
+    link: Link!
+    user: User!
+}
+
+type AuthPayload {
+    token: String
+    user: User
+}
+
+type Feed {
+    links: [Link!]!
+    count: Int!
+}
+
+enum LinkOrderByInput {
```

Comment:
Were you planning some order/sort by logic for the frontend?

Reply:
Awesome catch, yes I had in mind to do some more sorting options other than just by the number of votes.

Most useful might be to sort by newest posts since they will probably be buried at the bottom of the page if I only sort by votes. (since all new posts will have zero votes)